content/news/2024-06-30-stem-support.md

```diff
@@ -0,0 +1,34 @@
+title: "GSoC 2024: Adding STEM mixing support"
+authors: Antoine Colombier
+tags: gsoc, gsoc-2024, improvements
+slug: STEM-Mixing
+date: 2024-06-30 12:00:00
+
+Disclaimer: *The blog post primarily serves as the documentation for the [Google Summer of Code](https://summerofcode.withgoogle.com/) 2024 project: "Multi channel mixing support (STEMS)".*
+
+## Explore New Creative Horizons with Mixxx's Latest Feature: Stem Mixing
+
+We are excited to introduce a major new feature in Mixxx: Stem Mixing. This innovative tool, inspired by Native Instruments' concept from their Traktor Pro software, allows you to separate and manipulate different elements of a track in real-time. But what exactly are stems?
+
+Stems are individual audio tracks that make up a complete song. Typically, a song is split into four main stems: drums, bassline, harmony, and vocals. By isolating these elements, you can remix, mashup, and edit tracks on the fly, providing a new level of flexibility and creativity in your DJ sets.
+
+Mixxx now supports Native Instruments stem files, the current public specification for this format. Whether you're an amateur DJ eager to experiment with new techniques or a professional looking to enhance your performances, Stem Mixing in Mixxx offers powerful new capabilities to elevate your mixes.
+
+# Paving the Way for AI-Generated Stems
+
+The introduction of Stem Mixing in Mixxx is just the beginning of a revolutionary journey in DJing technology. This feature not only enhances your current capabilities but also sets the stage for future innovations, particularly in the realm of AI-generated stems. The Mixxx community is actively exploring ways to integrate artificial intelligence to create stems on the fly, pushing the boundaries of what’s possible in live DJ performances.
```

Comment:
Pushing boundaries ... Maybe you can mention one or two techniques explicit?

Reply:
Yep, that sounds like a good idea. With the live demo, I'm also planning to show some real examples too!